### PR TITLE
Send valid OpenID association session response

### LINKED ISSUE #2006: OpenID authentication fails due to malformed association session response
- CAS server version: 4.2.4
- Container/environment type/version: Apache Tomcat/8.0.32-gentoo

OpenID association fails during association phase because CAS server sends `%>` string as a successful association response. Authentication process is interrupted as Relaying Party cannot parse a malformed response.
[This JSP](https://github.com/apereo/cas/blob/4.2.x/cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp) is responsible for building a successful association response. It seems that commit fc82203bbe0534043039796670beee0185a3ede4 wiped out its contents. Restoring the previous file version fixes the issue.

I don't include CAS server logs as they don't seem to be relevant.

---
I have included the following in my report:

- [X] CAS server version
- [X] Container/environment type/version (Tomcat, Jetty, Java, Linux/Windows, etc)
- [X] Steps to duplicate the problem
- [X] Stacktraces and CAS logs where appropriate.
- [ ] Configuration/Overlay pushed to https://github.com/cas-projects/cas-issues/

### PR DESCRIPTION
**Pull request by marcmos**
Restores previous` casOpenIdAssociationSuccessView.jsp` contents (before cleanup).

Closes #2006.

### CODE REVIEW

File: cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp [hunk 1]
```diff
@@ -1 +1,28 @@
+<%--
```
Comment (mmoayyed) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp]: Remove the license header please.

**Comment by mmoayyed** [comment]
What was the previous version exactly? Where did u find it from?

**Comment by marcmos** [comment]
I just checked out this particular file to commit a4116d3b7971c389e250f74df87534d374b49ea7 (you can check its contents [here](https://github.com/apereo/cas/blob/a4116d3b7971c389e250f74df87534d374b49ea7/cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp)). I'm not sure whether this answers your question.

**Comment by marcmos** [comment]
I have just noticed that other files in this directory have `--%>` comment-closing tag left unbalanced. Don't you think those should be removed too?

File: cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp (hunk 1 above)
Comment (marcmos) [inline cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdAssociationSuccessView.jsp]: Removed in eebc2909ee75a40a2105c75c067b22015ada9f54.

**Comment by mmoayyed** [comment]
Yes those should be accounted for as well. Is there a way we could produce unit tests to account for the correctness of this patch in the future?

**Comment by mmoayyed** [comment]
@marcmos Going to merge. Thank you. When you find the time, please do submit another PR for the other files with the unbalanced closing tag.

**Pull request merged**